packages/react-core/src/components/DragDrop/Droppable.tsx

```diff
@@ -10,19 +10,29 @@ interface DroppableProps extends React.HTMLProps<HTMLDivElement> {
   zone?: string;
   /** Id to be passed back on drop events */
   droppableId?: string;
+  /** Don't wrap the component in a div. Requires passing a single child. */
+  noWrap?: boolean;
```

Comment:
noWrap is a css term - so I think you might need a clearer variable name - like noWrapper